scripts/nrel_solar.json

```diff
@@ -0,0 +1,1364 @@
+                        "size": 18
+                    },
+                    {
+                        "name": "GHI_DEC",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_JAN",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_FEB",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_MAR",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_APR",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_MAY",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_JUN",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_JUL",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_AUG",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_SEP",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_OCT",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_NOV",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    },
+                    {
+                        "name": "LATILT_DEC",
+                        "precision": 10,
+                        "type": "Real",
+                        "size": 18
+                    }
+                ]
+            }
+        },
+
+        {
+            "name" : "TMY3_NSRDB_Boundaries",
+            "path" : "nsrdb_boundary/nsrdb_bndry.shp",
```

Comment:
This path is incorrect.

Reply:
@ss-is-master-chief, this path does not exist "path" : "nsrdb_boundary/nsrdb_bndry.shp",
 "nsrdb_boundary/nsrdb_boundary/nsrdb_bndry.shp",